bazel_codegen/lib/src/summaries/resolvers.dart

```diff
@@ -57,51 +59,49 @@ class SummaryResolvers implements Resolvers {
   }
 }
 
-/// a [Resolver] backed by an [AnalysisContext].
+/// a [Resolver] backed by an [AnalysisDriver].
 class AnalysisResolver implements ReleasableResolver {
-  final AnalysisContext _analysisContext;
+  final AnalysisDriver _analysisDriver;
   final List<AssetId> _assetIds;
 
-  AnalysisResolver(this._analysisContext, this._assetIds);
+  AnalysisResolver(this._analysisDriver, this._assetIds);
 
   @override
-  void release() => _analysisContext.dispose();
+
+  /// Nothing to do in this impl, eventually [_analysisDriver.dispose()] should
+  /// be called but not until the driver is done being used.
+  void release() {}
 
   @override
-  bool isLibrary(AssetId assetId) {
+  Future<bool> isLibrary(AssetId assetId) async {
     var uri = assetUri(assetId);
-    var source = _analysisContext.sourceFactory.forUri2(uri);
-    return source != null &&
-        _analysisContext.computeKindOf(source) == SourceKind.LIBRARY;
+    // var source = _analysisDriver.sourceFactory.forUri2(uri);
```

Comment:
delete the commented out code.

Do we need a new way to handle missing sources or will `getSourceKind` not throw anymore?